backend/src/environment.ts

```diff
@@ -13,6 +14,9 @@ import plist from 'plist'
 import path from 'path'
 import os from 'os'
 import fs from 'fs'
+import lan from './LAN'
+import { environment } from '.'
+import EventBus from './EventBus'
```

Comment:
Let's move this stuff to the LAN class. Also line 18? You realize that's the same file that it's in... definitely don't need to do that.  ;)

Reply:
moved to lan class.  I think the IDE was trying to be helpful when I copied some code over.  missed that. removed

Reply:
👍